sdk/textanalytics/Azure.AI.TextAnalytics/README.md

```diff
@@ -17,9 +17,7 @@ Text Analytics is part of the Azure Cognitive Service for Language, a cloud-base
 [Source code][textanalytics_client_src] | [Package (NuGet)][textanalytics_nuget_package] | [API reference documentation][textanalytics_refdocs] | [Product documentation][language_service_docs] | [Samples][textanalytics_samples]
 
 ## Getting started
-
```

Comment:
I highly recommend installing the Microsoft Docs Pack (might be the Microsoft Learn Pack) now in VSCode. These are the same rules that learn.microsoft.com use and helps identify issues in the markdowns.

While this change isn't a problem rendering, it is a linting rule to keep the plaintext version more readable and ends up getting flagged, thus making actual issues harder to spot.

I try to keep all my markdowns clean with those rulesets for that reason: fix all the warnings - even seemingly lame ones like this - and the good warnings are easier to spot e.g., like not including locales e.g., "en-us", in links.

@weshaggard should we just add that VSCode extension pack into _.github/extensions.json_? I know I'm not the only one using them.

Reply:
I'm not opposed to you guys adding it. We have issue https://github.com/Azure/azure-sdk-tools/issues/1896 about MD linting in general but it hasn't been prioritized yet.

Reply:
It appears that it is now called the "Learn Authoring Pack". Ironically, I removed the blank lines because I personally thought that making it less "airy" made it easier to read in plain text. 😂 I undid that change and also fixed a couple of other pre-existing linting issues. Thanks!